app/controllers/autoyast_values.rb

```diff
@@ -0,0 +1,47 @@
+# Load values that will be used to populate the autoyast profile
+class AutoyastValues
```

Comment:
That's not a controller. I propose to move that to [lib/velum](https://github.com/kubic-project/velum/tree/master/lib/velum).

What does others like @ereslibre, @MaximilianMeister think about?